packages/contracts/package.json

```diff
@@ -15,11 +15,12 @@
   "license": "ISC",
   "devDependencies": {
     "ava": "^3.4.0",
+    "circomlib": "https://github.com/kobigurk/circomlib.git#347822604996bf25f659f96ee0f02810a1f71bb0",
```

Comment:
At the moment, the MiMC contract needs this specific fork of circomlib to compile.